provider/grpc.go

```diff
@@ -840,9 +840,18 @@ func (g *GRPCClient) GetSwapsStream(
 
 // GetNewRaydiumPoolsStream subscribes to a stream for getting recent swaps on projects & markets of interest.
 func (g *GRPCClient) GetNewRaydiumPoolsStream(
-	ctx context.Context,
+	ctx context.Context) (connections.Streamer[*pb.GetNewRaydiumPoolsResponse], error) {
```

Comment:
you should pass `includeCPMM` into here so that it is easier to test on the outside providers (i.e ws, grpc) instead of having to toggle it at the provider level

Reply:
But wouldn't that break existing clients as this is a public api?

I investigated go's support for optional parameters and parameters that have default values and saw that go does not support either of these. Hence, I went with the design pattern of adding in a new public api - GetNewRaydiumPoolsStreamV2 that adds in the ability to pass an includeCPMM parameter.

Is there a better design pattern for this?

Reply:
We don't do function overloads to support different signatures for different combination of flags in GO SDK. so I agree with @atulsriv

Reply:
Took out function overload. Added includeCPMM parameter to GetNewRaydiumPoolsStream.